openedx/tests/settings.py

```diff
@@ -5,8 +5,39 @@
 
 from __future__ import absolute_import, unicode_literals
 
+import sys
 import tempfile
 
+from path import Path
+
+# TODO: Remove the rest of the sys.path modification here and in (cms|lms)/envs/common.py
```

Comment:
Do we have a ticket for this work in our backlog?

Reply:
I didn't see one, so I just created it: [PLAT-1849](https://openedx.atlassian.net/browse/PLAT-1849)